### DISCUSSION TITLE
`bentoml.depends` on distributed services on kubernetes

### DISCUSSION THREAD

**Discussion by zsaladin** [bentoml/BentoML#4653] [Q&A]
There so many advantages to use `bentoml.depends`. 

I define multiple services in a service.py. And then I'd like to deploy each service into multiple Pods.

For example, Service `a`, `b` are defined in a single service.py. `a` service `bentoml.depends` on `b` service `a` service is deploy on A Pod. `b` Service is deploy on B Pod.

In this situation how does `a` service discover `b` service?

**Comment by antoniogomezalvarado** [comment]
I think this is a great topic as its not clear how `bentoml.depends` manifests in a Kubernetes setup
